src/frontend/src/icp-eth/components/tokens/ManageTokens.svelte

```diff
@@ -84,6 +88,23 @@
 		$pseudoNetworkChainFusion
 	]);
 
+	// To avoid strange behavior when the exchange data changes (for example, the tokens may shift
+	// since some of them are sorted by market cap), we store the exchange data in a variable during
+	// the life of the component.
+	let exchangesStaticData: ExchangesData | undefined;
+	onMount(() => {
```

Comment:
- While Svelte seems to be fine having multiple `onMount` declared, I would not be against having a single declaration / usage. Would you mind using a single `onMount`?

- Should we create a deep copy to even strenghness the fact that we do not want to dynamically update the list? `exchangesStaticData = nonNullish($exchanges) ? { ...$exchanges } : undefined;`

Reply:
- on it! do you prefer in this PR or another?
- makes sense